site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md

```diff
@@ -20,8 +20,7 @@ period (currently 30s) with a JSON body, and scale the target fleet based on the
 
 ### Prerequisites
 
-It is assumed that you have read the instructions to [Create a Game Server Fleet]({{< relref "./create-fleet.md" >}})
-and you have a running fleet of game servers or you could run command from Step #1.
+It is assumed that you have read the instructions to [Create a Game Server Fleet]({{< relref "./create-fleet.md" >}}).
```

Comment:
```suggestion
It is assumed that you have completed the instructions to [Create a Game Server Fleet]({{< relref "./create-fleet.md" >}}) and have a running fleet of game servers.
```

Or am wrong, do you not need the Fleet?

Reply:
Here we are creating the fleet in step 1 of this tutorial. So this tutorial a bit different to the `create-fleetautoscaler.md`.